Modules/_cursesmodule.c

```diff
@@ -170,34 +170,72 @@ class _curses.window "PyCursesWindowObject *" "&PyCursesWindow_Type"
 static PyObject *PyCursesError;
 
 /* Tells whether setupterm() has been called to initialise terminfo.  */
-static int initialised_setupterm = FALSE;
+static int CURSES_SETUPTERM_CALLED = FALSE;
 
 /* Tells whether initscr() has been called to initialise curses.  */
-static int initialised = FALSE;
+static int CURSES_INITSCR_CALLED = FALSE;
 
 /* Tells whether start_color() has been called to initialise color usage. */
-static int initialisedcolors = FALSE;
+static int CURSES_INITIALISED_COLORS = FALSE;
 
-static char *screen_encoding = NULL;
+static char *CURSES_SCREEN_ENCODING = NULL;
```

Comment:
> Are these 4 variables moved to the module state later?

No. Because I totally forgot about them in the other PR. But here, when I was isolating the components to select, I was like "hum, but those are actually global variables" and it confused me to have them in lowercase as if they were variables :(